src/apps/xpubs/xpubs.py

```diff
@@ -19,35 +19,41 @@ class XpubApp(BaseApp):
     prefixes = [b"fingerprint", b"xpub"]
 
     def __init__(self, path):
-        # we don't need to store anything
+        self.account = 0
         pass
 
     async def menu(self, show_screen, show_all=False):
         net = NETWORKS[self.network]
+        coin = net["bip32"]
         buttons = [
             (None, "Recommended"),
-            ("m/84h/%dh/0h" % net["bip32"], "Single key"),
-            ("m/48h/%dh/0h/2h" % net["bip32"], "Multisig"),
+            ("m/84h/%dh/%dh" % (coin, self.account), "Single key"),
+            ("m/48h/%dh/%dh/2h" % (coin, self.account), "Multisig"),
             (None, "Other keys"),
         ]
         if show_all:
-            coin = net["bip32"]
             buttons += [
-                ("m/84h/%dh/0h" % coin, "Single Native Segwit\nm/84h/%dh/0h" % coin),
-                ("m/49h/%dh/0h" % coin, "Single Nested Segwit\nm/49h/%dh/0h" % coin),
                 (
-                    "m/48h/%dh/0h/2h" % coin,
-                    "Multisig Native Segwit\nm/48h/%dh/0h/2h" % coin,
+                    "m/84h/%dh/%dh" % (coin, self.account),
+                    "Single Native Segwit\nm/84h/%dh/%dh" % (coin, self.account)
                 ),
                 (
-                    "m/48h/%dh/0h/1h" % coin,
-                    "Multisig Nested Segwit\nm/48h/%dh/0h/1h" % coin,
+                    "m/49h/%dh/%dh" % (coin, self.account),
+                    "Single Nested Segwit\nm/49h/%dh/%dh" % (coin, self.account)
+                ),
+                (
+                    "m/48h/%dh/%dh/2h" % (coin, self.account),
+                    "Multisig Native Segwit\nm/48h/%dh/%dh/2h" % (coin, self.account),
+                ),
+                (
+                    "m/48h/%dh/%dh/1h" % (coin, self.account),
+                    "Multisig Nested Segwit\nm/48h/%dh/%dh/1h" % (coin, self.account),
                 ),
             ]
         else:
-            buttons += [(0, "Show more keys"), (1, "Enter custom derivation")]
+            buttons += [(0, "Show more keys"), (2, "Change account number"), (1, "Enter custom derivation")]
         # wait for menu selection
-        menuitem = await show_screen(Menu(buttons, last=(255, None)))
+        menuitem = await show_screen(Menu(buttons, last=(255, None), title="Select the key"))
```

Comment:
Let's add the current account number to the menu note as well:
```suggestion
        menuitem = await show_screen(Menu(buttons, last=(255, None),
                                          title="Select the key",
                                          note="Current account number: %d" % self.account))
```